tagstudio/src/core/library.py

```diff
@@ -1114,24 +1053,16 @@ def refresh_dupe_files(self, results_filepath):
                         )
                 for match in matches:
                     # print(f'MATCHED ({match[2]}%): \n   {files[match[0]]} \n-> {files[match[1]]}')
-                    if os.name == "nt":
-                        file_1 = str(os.path.relpath(files[match[0]], self.library_dir))
-                        file_2 = str(os.path.relpath(files[match[1]], self.library_dir))
-                        if (
-                            file_1.lower() in self.filename_to_entry_id_map.keys()
-                            and file_2.lower() in self.filename_to_entry_id_map.keys()
-                        ):
-                            self.dupe_files.append(
-                                (files[match[0]], files[match[1]], match[2])
-                            )
-                    else:
-                        if (
-                            file_1 in self.filename_to_entry_id_map.keys()
-                            and file_2 in self.filename_to_entry_id_map.keys()
-                        ):
-                            self.dupe_files.append(
-                                (files[match[0]], files[match[1]], match[2])
-                            )
+                    file_1 = str(files[match[0]].relative_to(self.library_dir))
+                    file_2 = str(files[match[1]].relative_to(self.library_dir))
```

Comment:
you're converting the items of `files` into strings, but using `relative_to()` here assumes it should be instance of `Path` 🤔